include/z64save.h

```diff
@@ -1584,18 +1528,23 @@ typedef enum {
     /* 7 */ EVENTINF_DOG_RACE_STATE_MAX = 7
 } EventInfDogRaceState;
 
-#define GET_EVENTINF_DOG_RACE_STATE (gSaveContext.eventInf[0] & EVENTINF_DOG_RACE_STATE_MAX)
+#define GET_EVENTINF_DOG_RACE_STATE (gSaveContext.eventInf[EVENTINF_00] & EVENTINF_DOG_RACE_STATE_MAX)
 #define SET_EVENTINF_DOG_RACE_STATE(state) \
-    (gSaveContext.eventInf[0] = ((gSaveContext.eventInf[0] & (u8)~EVENTINF_DOG_RACE_STATE_MAX) | (state)))
+    (gSaveContext.eventInf[EVENTINF_00] =  \
+         ((gSaveContext.eventInf[EVENTINF_00] & (u8)~EVENTINF_DOG_RACE_STATE_MAX) | (state)))
 
-#define GET_EVENTINF_DOG_RACE_SELECTED_DOG_INDEX ((gSaveContext.eventInf[0] & (u8)~EVENTINF_DOG_RACE_STATE_MAX) >> 3)
+#define GET_EVENTINF_DOG_RACE_SELECTED_DOG_INDEX \
+    ((gSaveContext.eventInf[EVENTINF_00] & (u8)~EVENTINF_DOG_RACE_STATE_MAX) >> 3)
 #define SET_EVENTINF_DOG_RACE_SELECTED_DOG_INDEX(selectedDogIndex) \
-    (gSaveContext.eventInf[0] = (gSaveContext.eventInf[0] & EVENTINF_DOG_RACE_STATE_MAX) | (selectedDogIndex << 3))
+    (gSaveContext.eventInf[EVENTINF_00] =                          \
+         (gSaveContext.eventInf[EVENTINF_00] & EVENTINF_DOG_RACE_STATE_MAX) | (selectedDogIndex << 3))
 
 // This is exactly the same as the selected dog index; they never need both of them at the same time.
-#define GET_EVENTINF_DOG_RACE_RACE_STANDING ((gSaveContext.eventInf[0] & (u8)~EVENTINF_DOG_RACE_STATE_MAX) >> 3)
+#define GET_EVENTINF_DOG_RACE_RACE_STANDING \
+    ((gSaveContext.eventInf[EVENTINF_00] & (u8)~EVENTINF_DOG_RACE_STATE_MAX) >> 3)
 #define SET_EVENTINF_DOG_RACE_RACE_STANDING(raceStanding) \
-    (gSaveContext.eventInf[0] = (gSaveContext.eventInf[0] & EVENTINF_DOG_RACE_STATE_MAX) | (raceStanding << 3))
+    (gSaveContext.eventInf[EVENTINF_00] =                 \
+         (gSaveContext.eventInf[EVENTINF_00] & EVENTINF_DOG_RACE_STATE_MAX) | (raceStanding << 3))
```

Comment:
Same comment as before about the EVENTINF macros.

Reply:
And yeah, same thing here. The macros won't work for this one either.